app/controllers/spam2_controller.rb

```diff
@@ -21,7 +21,7 @@ def _spam
 
   def _spam_revisions
     if logged_in_as(%w(admin moderator))
-      @revisions = Revision.where(status: 0)
+      @revisions = Revision.where(status: 0).paginate(page: params[:page])
```

Comment:
Here, could `.paginate()` go on its own line aligned with `.where()` and `.order()`? Thanks!

Reply:
and the same for the other instances! This will be great, thanks!!!